.gitmodules

```diff
@@ -1,18 +1,18 @@
 [submodule "deps/restdown"]
 	path = deps/restdown
-	url = https://github.com/joyent/restdown.git
+	url = https://github.com/TritonDataCenter/restdown.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = https://github.com/joyent/javascriptlint.git
+	url = https://github.com/TritonDataCenter/javascriptlint.git
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = https://github.com/joyent/jsstyle.git
+	url = https://github.com/TritonDataCenter/jsstyle.git
 [submodule "deps/sdc-scripts"]
 	path = deps/sdc-scripts
 	url = https://github.com/joyent/sdc-scripts.git
```

Comment:
Any reason this one isn't getting updated too?

Reply:
Fixed.